app/core/views.py

```diff
@@ -648,16 +649,16 @@ def send_mail(self, politician):
         )
         profile_url_absolute = self.request.build_absolute_uri(profile_url)
         send_mail(
-            _('Freedomvote account link'),
-            _("""
+            unicode(_('Freedomvote account link')),
+            dedent(unicode(_("""
             Hello,
 
             You receive the link for your profile on Freedomvote:
```

Comment:
Wording: Recipients may or may not know about Freedmovote when they receive this email, so I would add some short introduction here.

Reply:
Yes they do. They just registrated themselves there...

Reply:
Okay, but still there should be a bit more explanation: "Keep this link and use it to login to your profile again" or similar

Reply:
Seems fair.